### PR TITLE
Caw 537

### PR DESCRIPTION
**Pull request by sherakama**
Adds much improved tab navigation and spoken feedback.

### CODE REVIEW

**Comment by sherakama** [comment]
@jbickar,

Please review. I have added a bunch of new aria information for the calendar block. You can see the functionality in action at: https://sites.stanford.edu/caw/events/month/2015-06

File: js/stanford_events_importer.js [hunk 1]
```diff
@@ -231,37 +231,36 @@
 // -----------------------------------------------------------------------------
 
   if(typeof $.fn.quiet_highlight !== "function") {
-      /**
+    /**
      * A quick yellow highlight that fades out.
      */
     $.fn.quiet_highlight = function() {
```
Comment (jbickar) [inline js/stanford_events_importer.js:237]: This is for the date/timepicker? If so, why is it in stanford_events_importer and not in stanford_date_timepicker?

Comment (sherakama) [inline js/stanford_events_importer.js:237]: There was a reason but it escapes me right now. I think it had to do with me not wanting to add the fade/highlight to the picker module but I suspect I did anyway. I will find out.

File: js/stanford_events_calendar.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      var block = $(".events-calendar-aria", context);
+      if ($(context).hasClass("events-calendar-aria")) {
+        block = context;
+        block.find(".date-heading a").focus();
+      }
+
+      if (block.length) {
+        var prev = block.find(".date-prev a");
+        var next = block.find(".date-next a");
+        block.attr("role", "navigation");
```
Comment (sherakama) [inline js/stanford_events_calendar.js]: Missing Drupal.t() here and other places.

**Comment by jbickar** [comment]
This looks (sounds) great to me. I'll leave it to you to add `Drupal.t()` as needed.

You should consider this as a patch to calendar.module.

**Comment by sherakama** [comment]
Yeah, I think I could generalize this one for the calendar module. That is a good idea.

File: js/stanford_events_importer.js (hunk 1 above)
Comment (sherakama) [inline js/stanford_events_importer.js:237]: This was not for the timepicker specifically but rather for the auto-population of one hour ahead on the to date and time.

**Pull request merged**